editor/src/clj/editor/gui.clj

```diff
@@ -486,10 +486,13 @@
   (properties/->choicebox (sort (remove empty? coll))))
 
 ;; SDK api
-(defn optional-gui-resource-choicebox [coll]
+(defn optional-gui-resource-choicebox
   ;; The coll will contain a "" entry representing "No Selection". Remove this
   ;; before sorting the collection. We then provide the "" entry at the top.
-  (properties/->choicebox (cons "" (sort (remove empty? coll)))))
+  ([coll]
+   (optional-gui-resource-choicebox coll sort))
+  ([coll custom-sort-fn]
+   (properties/->choicebox (cons "" (custom-sort-fn (remove empty? coll))) false)))
```

Comment:
This is SDK API, so I don't want to break it, instead I make it optional. So, now it's possible to specify a custom sort function